include/h_malloc.h

```diff
@@ -15,6 +15,11 @@ extern "C" {
 #define h_realloc realloc
 #define h_aligned_alloc aligned_alloc
 #define h_free free
+#if CONFIG_BLOCK_OPS_CHECK_SIZE && !defined(HAS_ARM_MTE)
+#define h_memcpy memcpy
+#define h_memmove memmove
+//#define h_memset memset
+#endif
```

Comment:
To simplify the code, something like `#define h_memcpy_real memcpy` would make sense I think, instead of having the condition in macros in the functions.

Reply:
that wouldn't work? they need to be called regardless of enabled or not